tests/test_functons.py

```diff
@@ -112,3 +118,80 @@ def test_distance_from_zero(test_distance_from_zero_input, test_distance_from_ze
 ])
 def test_square(distance_from_zero_type_input, distance_from_zero_type_expected):
     assert distance_from_zero_type(distance_from_zero_type_input) == distance_from_zero_type_expected
```

Comment:
@nicolas13sochi could you fix this?
`test_square` should be renamed to `distance_from_zero_type()`